src/ex5/server.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const { Sequelize } = require("sequelize");
+
+const sequelize = new Sequelize("ori", "root", "bgZONOT2602!", {
```

Comment:
it's worth keeping user and passwords in the .env file so it won't be part of your repo and open for everyone to see, also you don't need to initialize sequalize here.